schemas/telemetry/core/core.2.schema.json

```diff
@@ -32,6 +32,12 @@
     "arch": {
       "type": "string"
     },
+    "experiments": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
```

Comment:
Is there something we could do (maybe an option passed to jq) to get the object properties of the schema in sorted order?